### PR TITLE
Add pagination to quizzes

### PR DESCRIPTION
**Pull request by Mav-Ivan**
Add pagination to quizzes page

### CODE REVIEW

File: src/containers/my-quizzes/QuizzesContainer.tsx [hunk 1]
```diff
@@ -27,6 +29,7 @@ const TestsContainer = () => {
   const { openDialog } = useConfirm()
   const searchTitle = useRef<string>('')
   const breakpoints = useBreakpoints()
+  const { page, handleChangePage } = usePagination()
   const itemsPerPage = getScreenBasedLimit(breakpoints, itemsLoadLimit)
   const columnsToShow = ajustColumns(breakpoints, columns, removeColumnRules)
```
Comment (Tolik170) [inline src/containers/my-quizzes/QuizzesContainer.tsx:34]: I guess you should add skip: (page - 1) * itemsPerPage to getQuizzes use callback below

File: src/containers/my-quizzes/QuizzesContainer.tsx [hunk 2]
```diff
@@ -27,6 +29,7 @@ const TestsContainer = () => {
   const { openDialog } = useConfirm()
   const searchTitle = useRef<string>('')
   const breakpoints = useBreakpoints()
+  const { page, handleChangePage } = usePagination()
   const itemsPerPage = getScreenBasedLimit(breakpoints, itemsLoadLimit)
```
Comment (Tolik170) [inline src/containers/my-quizzes/QuizzesContainer.tsx:33]: I guess you should add:
skip: (page - 1) * itemsPerPage 
to getQuizzes use callback below

File: src/containers/my-quizzes/QuizzesContainer.tsx [hunk 3]
```diff
@@ -109,18 +112,25 @@ const TestsContainer = () => {
...
+          onRequestSort: () => null
+        }}
+        sx={styles.table}
+      />
+      <AppPagination
+        onChange={handleChangePage}
+        page={page}
+        pageCount={Math.ceil(response.count / itemsPerPage)}
+      />
+    </>
```
Comment (Yurenko) [inline src/containers/my-quizzes/QuizzesContainer.tsx:133]: To obtain new quizzes, you need to include a page in your request as the current method is ineffective.

File: src/containers/my-quizzes/QuizzesContainer.tsx (hunk 2 above)
Comment (Mav-Ivan) [reply src/containers/my-quizzes/QuizzesContainer.tsx:39]: done

File: src/containers/my-quizzes/QuizzesContainer.tsx (hunk 3 above)
Comment (Mav-Ivan) [reply src/containers/my-quizzes/QuizzesContainer.tsx:138]: done

**Review by Yurenko — APPROVED** [review]

**Review by Roman-Peretiatko — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Pull request merged**